src/Ionide.ProjInfo/Library.fs

```diff
@@ -30,7 +31,7 @@ module Init =
 /// In most cases you want to use `Ionide.ProjInf.WorkspaceLoader` type instead
 module ProjectLoader =
 
-    type LoadedProject = private LoadedProject of ProjectInstance
+    type LoadedProject = internal LoadedProject of ProjectInstance
```

Comment:
I made this internal so `WorkspaceLoaderViaProjectGraph` but I could move the logic into this module so this won't get exposed again.